Framework/Algorithms/src/SumOverlappingTubes.cpp

```diff
@@ -83,17 +84,23 @@ void SumOverlappingTubes::init() {
   declareProperty(make_unique<PropertyWithValue<bool>>("MirrorScatteringAngles",
                                                        false, Direction::Input),
                   "A flag to mirror the signed 2thetas. ");
+  declareProperty(make_unique<PropertyWithValue<bool>>("SplitCounts", false,
+                                                       Direction::Input),
+                  "A flag to split the counts between adjacent bins");
```

Comment:
Normally we avoid re-arranging properties to avoid breaking old stuff. However, I don't believe this is used directly by user at the moment, so perhaps we can move this to keep it with `ScatteringAngleTolerance`.

Could you also look at making `ScatteringAngleTolerance` an optional property that is only enabled if `SplitCounts` is checked? See `NormaliseToMonitor` for an example.

Reply:
Yes, this algorithm is not in use, apart from the reduction for D2B. So it is safe to reorder the properties to make more logical sense.